open-liberty/content.md

```diff
@@ -191,7 +191,7 @@ docker run -d -p 80:9080 -p 443:9443 \
 
 WebSphere Liberty is a commercial distribution of Open Liberty. There is an official docker image for websphere-liberty. The websphere-liberty docker image predates the open-liberty one, so to make it simpler to move from open-liberty to websphere-liberty (or vice versa) the images are broadly compatible. It should be possible to move from one to the other with a simple FROM clause change. Some considerations for moving between them:
 
-	Open Liberty installs into `/opt/ol` rather than `/opt/ibm`.
-	Use the `/config` folder for accessing the server configuration.
-	Use the `/output` folder for accessing the server output.
-	When adding your own SSL configuration use the `/config/configDropins/defaults/keystore.xml`.
+- Open Liberty installs into `/opt/ol` rather than `/opt/ibm`.
+- Use the `/config` folder for accessing the server configuration.
+- Use the `/output` folder for accessing the server output.
+- When adding your own SSL configuration use the `/config/configDropins/defaults/keystore.xml`.
```

Comment:
https://travis-ci.org/docker-library/docs/builds/560722551

This needs to be tabs instead of spaces (`-\t...`):

```diff
diff ./open-liberty/content.md markdownfmt/./open-liberty/content.md
--- /tmp/markdownfmt046124516
+++ /tmp/markdownfmt423016435
@@ -191,7 +191,7 @@
 
 WebSphere Liberty is a commercial distribution of Open Liberty. There is an official docker image for websphere-liberty. The websphere-liberty docker image predates the open-liberty one, so to make it simpler to move from open-liberty to websphere-liberty (or vice versa) the images are broadly compatible. It should be possible to move from one to the other with a simple FROM clause change. Some considerations for moving between them:
 
-- Open Liberty installs into `/opt/ol` rather than `/opt/ibm`.
-- Use the `/config` folder for accessing the server configuration.
-- Use the `/output` folder for accessing the server output.
-- When adding your own SSL configuration use the `/config/configDropins/defaults/keystore.xml`.
+-	Open Liberty installs into `/opt/ol` rather than `/opt/ibm`.
+-	Use the `/config` folder for accessing the server configuration.
+-	Use the `/output` folder for accessing the server output.
+-	When adding your own SSL configuration use the `/config/configDropins/defaults/keystore.xml`.
```

Reply:
I'm not sure why this is a tabs vs. spaces issue? Previously this paragraph was indented (using 4 spaces) and that made it look like a code block, but it should be a regular text block with a list of elements.

So I removed the code-block indenting and replaced each list item with `-` so it renders as a bullet-point list of items.

Reply:
Yes, a four space indent made it an unfenced code block and the removal of those spaces is correct.

Our markdown formatter requires unordered list items to be "dash tab" (`-\t`); using "dash space" (`- `) is not allowed.